### PR TITLE
Add Resident Evil: Revelations

### PR DESCRIPTION
**Pull request by eclenaeu**
Adds Resident Evil: Revelations.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/resident-evil-revelations/game.dat [hunk 1]
```diff
@@ -0,0 +1,34 @@
+title = "Resident Evil: Revelations"
+description = "Resident Evil: Revelations, is a third-person shooter video game, starring original series leads Chris Redfield and Jill Valentine, the game displays hallways reminiscent of the mansions of the original Resident Evil and Resident Evil 5. While most of the game takes place aboard the luxurious cruiser Queen Zenobia, players also briefly visit other settings."
```
Comment (BreadFish64) [inline games/resident-evil-revelations/game.dat]: maybe add the publisher, Capcom, somewhere in here

Comment (Schplee) [reply games/resident-evil-revelations/game.dat]: Although your description is worded well, the description should be just one or two sentences as describe in our [Contributions Guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md).

File: games/resident-evil-revelations/game.dat [hunk 2]
```diff
@@ -0,0 +1,34 @@
+title = "Resident Evil: Revelations"
+description = "Resident Evil: Revelations, is a third-person shooter video game, created by Capcom, starring original series leads Chris Redfield and Jill Valentine, the game displays hallways reminiscent of the mansions of the original Resident Evil and Resident Evil 5. While most of the game takes place aboard the luxurious cruiser Queen Zenobia, players also briefly visit other settings."
```
Comment (Leo626) [inline games/resident-evil-revelations/game.dat]: The genre should be survival horror (according to Wikipedia).

Comment (eclenaeu) [reply games/resident-evil-revelations/game.dat]: you said well, the genre, not the style of play.

Comment (Schplee) [reply games/resident-evil-revelations/game.dat]: I think the description is sufficient as is.

Comment (Leo626) [reply games/resident-evil-revelations/game.dat]: This is more of a nitpick. If no one really has a problem with this, then I guess it LGTM.

**Pull request merged**